Neos.ContentRepositoryRegistry/Classes/Command/ContentCommandController.php

```diff
@@ -114,7 +114,7 @@ public function moveDimensionSpacePointCommand(string $source, string $target, s
 
         $contentRepositoryInstance->handle(
             MoveDimensionSpacePoint::create(
-                $workspaceInstance->currentContentStreamId,
```

Comment:
we wouldnt event actually need to resolve the workspace instance here but i guess it leads to better errors if we do so.